### PR TITLE
Create GooglePlusButtonByRakesh-116.html

### PR DESCRIPTION
**Pull request by Rakesh-116**

### CODE REVIEW

File: modules/buttons/GooglePlusButtonByRakesh-116.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        border-width: 0;
+        border-radius:2px;
+        color:#ffffff;
+        font-size:15px;
+        font-weight: 100;
+      }
+    </style>
+    <div>
+        <button class="google-button">
+            <img src="Google_Plus_logo_(2015-2019).svg.png"/>Google +
```
Comment (NtemKenyor) [inline modules/buttons/GooglePlusButtonByRakesh-116.html]: I can't seem to find this image location? Did you actually need it? 

**Story Time**
In this "Session of Buttons", I saw a guy who designed the entire button without an svg or any img. I was wowed. Just some Text span and nice styling. lol....

**Review by NtemKenyor — CHANGES_REQUESTED** [review]
Please make the changes and help improve the code.

File: modules/buttons/GooglePlusButtonByRakesh-116.html (hunk 1 above)
Comment (Rakesh-116) [reply modules/buttons/GooglePlusButtonByRakesh-116.html]: I have done the changes according to your query..

**Review by NtemKenyor — APPROVED** [review]
thank you...

**Comment by Rakesh-116** [comment]
Thank you for your response

**Pull request merged**